util/templateutil.go

```diff
@@ -19,7 +19,10 @@ import (
 	"github.com/kubernetes/deployment-manager/common"
 )
 
-var re = regexp.MustCompile("github.com/(.*)/(.*)/(.*)/(.*):(.*)")
+var RE = regexp.MustCompile("github.com/(.*)/(.*)/(.*)/(.*):(.*)")
```

Comment:
Since these are public vars now, probably call them something better like TemplateRegistryMatcher and PackageRegistryMatcher